source/extensions/quic_listeners/quiche/active_quic_listener.cc

```diff
@@ -112,14 +117,37 @@ void ActiveQuicListener::onData(Network::UdpRecvData& data) {
                                   /*packet_headers=*/nullptr, /*headers_length=*/0,
                                   /*owns_header_buffer*/ false);
   quic_dispatcher_->ProcessPacket(self_address, peer_address, packet);
+
+  if (!schedule_process_buffered_enabled_) {
```

Comment:
Shouldn't this be equivalent to:
if (!schedule_process_buffered_->enabled()) {
  schedule_process_buffered_->scheduleCallbackNextIteration();
}

Reply:
Yes; not sure how I missed that when looking at the API.

Reply:
After #13097 you should be able to unconditionally call scheduleCallbackNextIteration